react-native/react/common-adapters/terminal.desktop.js

```diff
@@ -0,0 +1,31 @@
+import React, {Component} from '../base-react'
+import {globalStyles, globalColors} from '../styles/style-guide'
+
+export default class Terminal extends Component {
```

Comment:
Create a `terminal.js.flow` for this too (it can have empty props, but that way flow will know it's a react component)